lib/generators/administrate/dashboard/dashboard_generator.rb

```diff
@@ -10,7 +10,7 @@ class DashboardGenerator < Rails::Generators::NamedBase
         enum: "Field::String",
         float: "Field::Number",
         integer: "Field::Number",
-        time: "Field::DateTime",
```

Comment:
In my opinion yes, because when we create a time attribute we want the `Field::Time` instead of the current `Field::DateTime`